azure-pipelines.perf-test.yml

```diff
@@ -21,7 +21,16 @@ jobs:
         displayName: yarn
 
       - script: |
-          yarn check-if-v0-v8-v9-affected
+          NorthstarAffected=$(node ./scripts/monorepo/checkIfPackagesAffected.js --@fluentui/react-northstar)
+          V8Affected=$(node ./scripts/monorepo/checkIfPackagesAffected.js --@fluentui/react --@fluentui/react-components)
```

Comment:
Needed to have the explicit command here since doing `V8Affected=$(yarn check-if-package-affected --@fluentui/react --@fluentui/react-components)` results in a funky output below:
![image](https://user-images.githubusercontent.com/8649804/155633381-9d9b9167-3cef-4ff5-82e6-2dc8a97dab1a.png)

Reply:
@TristanWatanabe  you can remove those yarn logs with `yarn --silent check-if-v0-v9-v9-affected`

Reply:
question is if we need to add another npm script alias at all. invoking this directly via node would not produce `yarn` specific churn

Reply:
ah you already went the node route - awesome :D